### ISSUE TITLE
declxml on Arch Linux

### ISSUE THREAD

**Issue by clintval** [gatkin/declxml#4]
Hi @gatkin. Great project! I started using `declxml` this week and hope to contribute in the future.

I wanted to let you know your project is available on Arch Linux platforms: 
https://aur.archlinux.org/packages/python-declxml-git/

**Comment by gatkin** [comment]
Awesome, thank you doing that! What needs to be done to make sure the Arch Linux package stays up-to-date whenever we make a release to PyPi? Is that a manual process?

**Comment by clintval** [comment]
I noticed you just made a release! Congrats!

The package I created (`python-declxml-git`) tracks the `master` branch and can be updated by the maintainer (me!) whenever is convenient. This allows users to sign-up for a bleeding edge install.

Now that you have a release I will create a package `python-declxml` which will be updated every PyPi release. This process is manual and is typical for other rolling-release packages.

Arch users will have the choice to install the rolling release package or the bleeding edge package. By creating these packages on the AUR I have volunteered to keep them up to date. I am now *watching* your package on GitHub for updates.

Feel free to close this issue if this is OK with you! Just wanted to share the update!
